sigle/src/modules/layout/components/AppHeader.tsx

```diff
@@ -42,8 +48,11 @@ const StatusDot = styled('div', {
 
 export const AppHeader = () => {
   const { user } = useAuth();
-  const router = useRouter();
+  const router = useRouter() || { query: { username: '' } };
```

Comment:
I think here it would be better to create a global mock like this https://github.com/vercel/next.js/issues/7479#issuecomment-533657701.

That way all the future tests would benefit from the fix too. Also the code itself should not contain special code related to the tests :)

Reply:
Ah I see! That makes sense. Where would be the best place to add the global mock?

Reply:
You can do like this https://jestjs.io/docs/manual-mocks#mocking-node-modules

Reply:
Great, thanks :)